dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+click==7.1.2
```

Comment:
If we think this will be a commonly used feature then we should add it directly to requirements or create a cli-requirements, which I saw you have already done. I would keep it out of this one not to be duplicated. IMO dev-requirements is for developer-only dependencies but command line will be used by normal users.

Reply:
We need to add it here in order for the CI for the tests to pass. Adding it to dev-requirements.txt is what we currently do for all of the libraries that have the `[cli]` install option.

If you want, we can get rid of the `[cli]` install option and just add `click` to requirements.txt but I figured the current implementation is more consistent with the rest of the libraries.

Reply:
We can change this line to `pip install .[cli]` and it should install click as needed but it's not really a major issue. What we have here is also fine with me.